test/e2e/testing-manifests/storage-csi/mock/csi-mock-driverinfo.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
We can do it as a follow-on, but it would be nice to enable the test for hostpath driver too

Reply:
This is on my todo list. It cannot be done at the moment because we first need to merge this PR, then do a release of the hostpath driver with a driver info that specifies both modes.

Reply:
Let me clarify: I was thinking of the kubernetes-csi Prow jobs. For the Kubernetes E2E, testing with the hostpath driver gets enabled in ab6c7bb080467e23f5eb3dfb0c409c6316305cd3, which is part of this PR.